### PR TITLE
.github: Test in Fedora 34

### PR DESCRIPTION
**Pull request by jjardon**

### CODE REVIEW

**Comment by gtristan** [comment]
This still refers to a non-existing fedora-34 container image :-/

File: .github/compose/ci.docker-compose.yml [hunk 1]
```diff
@@ -22,10 +22,11 @@ x-tests-template: &tests-template
...
 
-  fedora-32:
+  fedora-33:
     <<: *tests-template
+    image: registry.gitlab.com/buildstream/buildstream-docker-images/testsuite-fedora:33-${CI_IMAGE_VERSION:-latest}
 
-  fedora-33:
+  fedora-34:
     <<: *tests-template
     image: registry.gitlab.com/buildstream/buildstream-docker-images/testsuite-fedora:33-${CI_IMAGE_VERSION:-latest}
```
Comment (juergbi) [inline .github/compose/ci.docker-compose.yml]:
```suggestion
    image: registry.gitlab.com/buildstream/buildstream-docker-images/testsuite-fedora:34-${CI_IMAGE_VERSION:-latest}
```

Comment (jjardon) [reply .github/compose/ci.docker-compose.yml]: Oh, of course, thanks @juergbi

**Comment by gtristan** [comment]
Since you asked us to rebase this, I've done so and opened #1532 in favor of this one (holding the same commits from @jjardon)

**Pull request closed without merging**